projects/web-app/src/components/app-shell/app-header.vue

```diff
@@ -31,12 +36,20 @@ export default {
   left: 0;
   width: 100%;
   height: var(--header-height);
+}
+
+.app-header__left {
```

Comment:
I think `.app-header__section` or `.app-header__column` with a `--left` or `--right` modifier class is more BEM-like I think.

Reply:
👌